### PR TITLE
DEV-655 awsprocesscreds-saml patch

### PR DESCRIPTION
**Pull request by asaranglr**
*Issue*
MFA seems to stall when using Okta and Google Authenticator app

*Description of changes:*
Get rid of saml assertion functionality from mfa_push
Update ambiguous user prompts

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsprocesscreds/saml.py [hunk 1]
```diff
@@ -478,21 +476,6 @@ def retrieve_saml_assertion(self, config):
                 raise SAMLError(self._ERROR_MFA_ENROLL)
             elif parsed["status"] == "MFA_REQUIRED":
                 return self.process_mfa_verification(endpoint, parsed)
-        # If we get to here, the chances are we're running the functional
```
Comment (joshk0) [inline awsprocesscreds/saml.py]: Why was this removed?

**Review by joshk0 — COMMENTED** [review]
Looks like Travis failed due to `flake8`

```
flake8 --ignore=E731,W503 --exclude compat.py --max-complexity 10 awsprocesscreds/
awsprocesscreds/cli.py:58:80: E501 line too long (93 > 79 characters)
awsprocesscreds/saml.py:232:11: E111 indentation is not a multiple of four
```

Also, recommend putting your description into the commit message, rather than the PR (though I guess if you end up squashing and merging, that will become the new commit message.) The original commit message is 'awsprocesscreds-saml patch', which isn't super helpful.

**Comment by joshk0** [comment]
still failing ^

```
###### FLAKE8 #####
# No unused imports, no undefined vars
flake8 --ignore=E731,W503 --exclude compat.py --max-complexity 10 awsprocesscreds/
flake8 tests/unit/ tests/functional/ tests/integration/
tests/unit/test_saml.py:353:80: E501 line too long (90 > 79 characters)
tests/unit/test_saml.py:357:1: E302 expected 2 blank lines, found 1
```

**Comment by asaranglr** [comment]
oops. didn't see the 2nd flake8 line in the output. All checks passed.
PTAL @joshk0

**Comment by joshk0** [comment]
I hate to break it to you but it looks like it still failed
https://travis-ci.com/LiveRamp/awsprocesscreds/jobs/126108708

**Comment by asaranglr** [comment]
Praise the almighty! PTAL Josh

**Review by joshk0 — APPROVED** [review]
LGTM. I saw there was a change involving the LiveRamp network which I definitely wouldn't want to merge upstream. Assuming it's gone, ship this and I would consider seeing how the other PR is going upstream and adding this change on since it would help others too.

**Comment by asaranglr** [comment]
Ohh, I removed that from this change. prompting for authentication solves the issue. Thank you.
